docs/source/user_guide.rst

```diff
@@ -155,6 +155,42 @@ Alternatively, we can add our custom pass, as shown in the following example.
       circuit_to_compile, custom_passes=[MyCustomPass()]
    )
 
+An example of a custom pass: Approximate Quantum Compilation via MPS encoding
+=============================================================================
+The ``MPSEncoder`` is a custom pass provided in ``ucc.aqc``. Users can opt for `qmprs<https://github.com/Qualition/qmprs>`_ for a more advanced implementation of the same pass.
```

Comment:
By opt for, do you mean they would need to also install that package? If so, please be explicit about that instruction and how to do it.